registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -216,6 +217,7 @@
   dmap:
     "sha256:6cb237a6445f64afb7ed92517c79fcf3a63b02e20a28aea3fc4274f9aeffd6d1": ["v1.10.0-beta.0"]
     "sha256:0bd716555d228c055bc2210b81e3276575d989a8f12c60060da269360536aab9": ["v1.10.0-beta.1"]
+    "sha256:21a6988bf23772dd6fa61ec57f36e66b2180ceb282fe767ef6a567dfae02e54d": ["v1.10.0-rc.0"]
```

Comment:
`21a6988bf237` matches the clusterctl [staging repo for v1.10.0-rc.0](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:21a6988bf23772dd6fa61ec57f36e66b2180ceb282fe767ef6a567dfae02e54d/details?invt=AbuOLA)